packages/core/test/VNodes.test.ts

```diff
@@ -1146,6 +1147,32 @@ describe('core', () => {
                     });
                 });
             });
+            describe('Custom VNode', () => {
+                it('should create and parse a custom node', () => {
+                    const editor = new JWEditor();
+                    class MyCustomNode extends VNode {
+                        customKey = 'yes';
+                        constructor() {
+                            super('custom');
+                        }
+                        static parse(node: Node): VNode | VNode[] | null {
```

Comment:
The logic of the parse should not be in VNode. It is strange if the vnode has to parse and create other VNode types.